### PR TITLE
Prepare for refresh SDKs in bulk to release the extension mocking function - 5

### PR DESCRIPTION
**Pull request by ArthurMa1978**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/managednetworkfabric/Azure.ResourceManager.ManagedNetworkFabric/CHANGELOG.md [hunk 1]
```diff
@@ -1,20 +1,21 @@
 # Release History
 
-## 1.2.0-beta.1 (Unreleased)
+## 1.1.1 (2023-11-29)
```
Comment (live1206) [inline sdk/managednetworkfabric/Azure.ResourceManager.ManagedNetworkFabric/CHANGELOG.md:3]: should this be 1.2.0?

File: sdk/marketplace/Azure.ResourceManager.Marketplace/CHANGELOG.md [hunk 2]
```diff
@@ -1,15 +1,15 @@
 # Release History
 
-## 1.2.0-beta.1 (Unreleased)
+## 1.1.1 (2023-11-29)
```
Comment (live1206) [inline sdk/marketplace/Azure.ResourceManager.Marketplace/CHANGELOG.md:3]: should this be 1.2.0?

File: sdk/mobilenetwork/Azure.ResourceManager.MobileNetwork/CHANGELOG.md [hunk 3]
```diff
@@ -1,15 +1,15 @@
 # Release History
 
-## 1.2.0-beta.1 (Unreleased)
+## 1.1.1 (2023-11-29)
```
Comment (live1206) [inline sdk/mobilenetwork/Azure.ResourceManager.MobileNetwork/CHANGELOG.md:3]: should this be 1.2.0?

File: sdk/netapp/Azure.ResourceManager.NetApp/CHANGELOG.md [hunk 4]
```diff
@@ -1,29 +1,32 @@
 # Release History
 
-## 1.5.0-beta.1 (Unreleased)
+## 1.4.2 (2023-11-29)
```
Comment (live1206) [inline sdk/netapp/Azure.ResourceManager.NetApp/CHANGELOG.md:3]: should this be 1.5.0?

File: sdk/networkcloud/Azure.ResourceManager.NetworkCloud/CHANGELOG.md [hunk 5]
```diff
@@ -1,15 +1,15 @@
 # Release History
 
-## 1.1.0-beta.1 (Unreleased)
+## 1.0.1 (2023-11-29)
```
Comment (live1206) [inline sdk/networkcloud/Azure.ResourceManager.NetworkCloud/CHANGELOG.md:3]: should it be 1.1.0?

File: sdk/newrelicobservability/Azure.ResourceManager.NewRelicObservability/CHANGELOG.md [hunk 6]
```diff
@@ -1,15 +1,15 @@
 # Release History
 
-## 1.1.0-beta.1 (Unreleased)
+## 1.0.1 (2023-11-29)
```
Comment (live1206) [inline sdk/newrelicobservability/Azure.ResourceManager.NewRelicObservability/CHANGELOG.md:3]: should it be 1.1.0?

File: sdk/nginx/Azure.ResourceManager.Nginx/CHANGELOG.md [hunk 7]
```diff
@@ -4,12 +4,12 @@
 
 ### Features Added
 
-### Breaking Changes
-
-### Bugs Fixed
+- Enable mocking for extension methods, refer this [document](https://aka.ms/azsdk/net/mocking) for more details.
```
Comment (live1206) [inline sdk/nginx/Azure.ResourceManager.Nginx/CHANGELOG.md:7]: no version or release date update?

File: sdk/peering/Azure.ResourceManager.Peering/CHANGELOG.md [hunk 8]
```diff
@@ -1,15 +1,15 @@
 # Release History
 
-## 1.3.0-beta.1 (Unreleased)
+## 1.2.1 (2023-11-29)
```
Comment (live1206) [inline sdk/peering/Azure.ResourceManager.Peering/CHANGELOG.md:3]: should it be 1.3.0?

File: sdk/nginx/Azure.ResourceManager.Nginx/CHANGELOG.md (hunk 7 above)
Comment (live1206) [reply sdk/nginx/Azure.ResourceManager.Nginx/CHANGELOG.md:7]: Also, this should be located in `Features added`

File: sdk/managednetworkfabric/Azure.ResourceManager.ManagedNetworkFabric/CHANGELOG.md (hunk 1 above)
Comment (ArthurMa1978) [reply sdk/managednetworkfabric/Azure.ResourceManager.ManagedNetworkFabric/CHANGELOG.md:3]: A minor version bump occurs only when the api-version is upgraded.

File: sdk/nginx/Azure.ResourceManager.Nginx/CHANGELOG.md (hunk 7 above)
Comment (ArthurMa1978) [reply sdk/nginx/Azure.ResourceManager.Nginx/CHANGELOG.md:7]: There is another ngix pr working on the upgrade https://github.com/Azure/azure-sdk-for-net/pull/40201

**Review by live1206 — APPROVED** [review]

**Pull request merged**